flask_nemo/__init__.py

```diff
@@ -64,10 +67,17 @@ class Nemo(object):
     :type css: [str]
     :param js: Path to additional javascripts to load
     :type js: [str]
-    :param templates: Register or override templates (Dictionary of index / path)
+    :param templates: Register or override templates (Dictionary of namespace / directory containing template)
     :type templates: {str: str}
     :param statics: Path to additional statics such as picture to load
     :type statics: [str]
+    :param prevent_plugin_clearing_assets: Prevent plugins to clear the static folder route
+    :type prevent_plugin_clearing_assets: bool
+    :param original_breadcrumb: Use the default Breadcrumb plugin packaged with Nemo (Default: True)
```

Comment:
I think it might be better to do this more generally because what if additional default plugins are added over time? maybe it would be better to pass a dict?